js/modules/pattern-fill.src.js

```diff
@@ -199,6 +282,23 @@ wrap(H.Series.prototype, 'getColor', function (proceed) {
 });
 
 
+/**
+ * Set default pattern dimensions on points that have their own color pattern
+ */
+wrap(H.Series.prototype, 'render', function (proceed) {
```

Comment:
Shouldn't we use `render` event, to limit number `wrap`'s?

Reply:
Yes ideally, but issues arise with series types that already wrap render. We need this to run before render, but the `render` event happens in `Series.render`, so anything that wraps before render will mess it up. In order for the event system to work we have to move away from wraps entirely in these cases - which I think is the intention eventually - and have a proper priority queue for events.